gateway-ha/src/test/java/io/trino/gateway/ha/security/LbLdapClientTest.java

```diff
@@ -135,7 +135,7 @@ public void testMemberof()
         String ret = lbLdapClient.getMemberOf(user);
 
         log.info("ret is %s", ret);
-        assertThat(ret).isEqualTo("Admin,User");
+        assertThat(ret).isEqualTo("ADMIN_USER");
```

Comment:
It could if this in LoginResource was also refactored to process a comma delimiter

https://github.com/trinodb/trino-gateway/blob/739ec5e67799dd5df0ab8aaec7934e21a63507fa/gateway-ha/src/main/java/io/trino/gateway/ha/resource/LoginResource.java#L110

Reply:
I saw `.split("_")` being hard coded everywhere. Let's stay in `_` for now.